src/app/components/ReadTime/index.tsx

```diff
@@ -88,4 +107,33 @@ const ReadTime = ({
   );
 };
 
-export default ReadTime;
+export const ReadTimeHomepage = ({
```

Comment:
Shall we make this generic so that it could potentially be used elsewhere - not on a Homepage e.g. Onward Journeys?

```suggestion
export const ReadTime = ({
```

Reply:
Yep sounds good. My thought process making separate article/ homepage components is that it would be really clear to devs where they are being used.

But this approach lines us up better for simplifying the component and only having one export. We also then have the option to delete the article component as part of the tear down, and not need to rename this one.